### PR TITLE
Allow to use newer version of faraday gem

### PR DESCRIPTION
**Pull request by paduszynski**
we need new features from faraday
I looked into code and cannot see anything that could broke here based on faraday changes

### CODE REVIEW

File: whatsapp_sdk.gemspec [hunk 1]
```diff
@@ -46,7 +46,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency('sorbet')
   spec.add_dependency('sorbet-runtime')
 
-  spec.add_dependency("faraday", "~> 2.3.0")
+  spec.add_dependency("faraday", ['>= 2.3.0', '< 3.0'])
```
Comment (ignacio-chiazzo) [inline whatsapp_sdk.gemspec:49]: Could you remove the higher version? The usage of this is minimal so it should be very low risk to have no max version.

**Review by ignacio-chiazzo — COMMENTED** [review]
Hey, thanks for the changes! 

The usage of `faraday` in the gem is very minimal, only to make basic http queries on the Graph API. I am not concern of making >=, but I would like us to remove the cap.

File: whatsapp_sdk.gemspec (hunk 1 above)
Comment (paduszynski) [reply whatsapp_sdk.gemspec]: yeah should be fine now

File: Gemfile.lock [hunk 2]
```diff
@@ -2,7 +2,7 @@ PATH
   remote: .
   specs:
     whatsapp_sdk (0.6.0)
-      faraday (~> 2.3.0)
+      faraday (>= 2.3.0, < 3.0)
```
Comment (ignacio-chiazzo) [inline Gemfile.lock:5]: Could you update the `gemfile.lock` too?

**Comment by ignacio-chiazzo** [comment]
Thanks for the PR!

**Pull request merged**